Providers/nxOMSAutomationWorker/automationworker/worker/main.py

```diff
@@ -66,7 +66,7 @@ def get_worker_and_sandbox_processes(process_list):
         """
         worker_and_sandbox_processes = []
         for process in process_list:
-            if "python" in process.cmd and ("hybridworker.py" in process.cmd or "sandbox.py" in process.cmd) \
+            if "python2" in process.cmd and ("hybridworker.py" in process.cmd or "sandbox.py" in process.cmd) \
```

Comment:
for this release we will also have to consider "python" because the old version of our module would have started the process as "python <hybridworker.py>", when the version updates this will not stop the processes started by the older version.

Reply:
why were we checking just for python and not python2 here (like in case of 3.x scripts)?

Reply:
Changes are pushed as per discussion